azure/v14.1.0/README.md

```diff
@@ -0,0 +1,59 @@
+# :zap: Giant Swarm Release v14.1.0 for Azure :zap:
+
+This is the first workload cluster release to support [spot VMs](https://azure.microsoft.com/en-us/pricing/spot/) on Azure.
+
+With this release, the `volumeBindingMode` for the following `storageClasses` will be changed to `WaitForFirstConsumer`:
+
+- default
+- managed-premium
+- managed-standard
+
+We made this change to allow kubernetes making better decisions about which availability zone to create new
+`AzureDisks` into, according to scheduling of pods.
+
+This will not affect any `PersistentVolume` already existing, but will affect future provisioning.
```

Comment:
```suggestion
This will not affect any `PersistentVolume` already existing, but will affect future provisioning. In case of old behavior being preferable, creation of additional storageclass will be required.
```